Samples/Swift/DaysUntilBirthday/Shared/ViewModels/VerifiedAgeViewModel.swift

```diff
@@ -32,12 +41,31 @@ final class VerifiedAgeViewModel: ObservableObject {
   /// Creates an instance of this view model.
   init() {
     self.verificationState = .unverified
+    self.ageVerificationSignal = "Signal Unavailable"
```

Comment:
Can this be an enum with a `String` raw value?

Reply:
Made `enum AgeVerificationSignal` to hold this value.